qiita_pet/static/js/networkVue.js

```diff
@@ -805,10 +866,20 @@ Vue.component('processing-graph', {
     // if the graph is not available
     vm.updateGraph();
     setInterval(function() {
-      // Only update if the graph has not been generated yet
-      if (vm.poll) {
-        vm.updateGraph();
+      vm.countdownPoll -= 1;
+      $('#countdown-span').html(vm.countdownPoll);
+      if (vm.countdownPoll === 0) {
+        // Reset the counter for every 15 seconds
+        vm.countdownPoll = 15;
+
+        // Check for the initial poll - it only happens if a graph doesn't exist yet
+        if (vm.initialPoll) {
+          vm.updateGraph();
+        }
+
+        // Update the status of the jobs
+        vm.update_job_status();
       }
-    }, 5000);
+    }, 1000);
```

Comment:
Every second, gulp!

Reply:
Note that although this call is performed every second, the actual update only happens every 15 seconds. The every second is needed if you want the countdown to inform the user when the interface is going to be updated, as it can be seen in the GIF (look just above the network graph).

Reply:
Forgot to mention that in the GIF the counter is set to reset the interface every 5 seconds, but that was for testing, the actual code does it every 15 seconds.